functions/methods.py

```diff
@@ -1,33 +1,50 @@
-from typing import List
+import datetime
 
 
-def read_the_file(file_path: str):
-    with open(file_path, 'r') as file:
-        data = file.readlines()
-        if data:
-            print('file_content:')
-            for line in data:
-                process_the_content(line.strip())
-        else:
-            print('file_is_empty.')
+def read_the_file(file_path: str) -> list:
+    try:
+        with open(file_path, 'r') as file:
+            data = file.readlines()
 
+        if not data:
+            print('File is empty.')
+        return data
 
-def calculate_average(scores: List[float]):
-    if len(scores) > 0:
+    except FileNotFoundError:
+        print('File not found.')
+
+
+def process_the_content(lines: list) -> list[dict]:
+    results = []
+    for line in lines:
+        fields = line.strip().split(',')
+        username = fields[0]
+        scores = []
+        for score in fields:
+            try:
+                score_value = float(score.strip().replace(',', '', 1))
+                scores.append(score_value)
+            except ValueError:
+                pass
```

Comment:
This `for` loop for parsing input data should be moved to `read_the_file` as it covers file content validation and parsing into Python structures